src/App.js

```diff
@@ -4,40 +4,27 @@ import './App.css';
 
 const App = () => {
 
-const api_key = 'EW43splo4ni8vWJ6oN2cJunf49yeRxy6IFk4aOTb';
-const[search, setSearch] = useState('');
-const [data, setdata] = useState([]);
-
-// const [query, setQuery] = useState('yello');
-
-
-useEffect(() => {
-  getParks();
-
-});
-
-const getParks = async () => {
-  const response = await fetch(`https://developer.nps.gov/api/v1/parks?parkCode=acad&api_key=${api_key}`)
-  // const response = await fetch (`https://developer.nps.gov/api/v1/activities?id=hiking&q=q&limit=20&start=0&api_key=EW43splo4ni8vWJ6oN2cJunf49yeRxy6IFk4aOTb`)
-  const data = await response.json();
-  
-  
-  
-  console.log(data);
-  
-  
-};
-
-const updatedSearch = e =>{ 
-  setSearch(e.target.value);
-
-}
-// const getSearch = e => {
-//   e.preventDefault();
-//   setQuery(search);
-//   getSearch('');
-// }
-
+  // const api_key = 'EW43splo4ni8vWJ6oN2cJunf49yeRxy6IFk4aOTb';
+  const api_key = 'iBIDgu279HOLibT9B72MGBhW3NOT8PIbD92dfSIr';
```

Comment:
There was some type of loop happening in the previous code that made it keep requesting from the API, and so I hit an API request limit using your token (😮  sorry!), so I replaced this with a new token I made.

Your token may still give errors because of the limit, but that should go away in an hour or so (feel free to use mine in the meantime)